drafts/developer-guide/pages/data-privacy/data-privacy-console.adoc

```diff
@@ -0,0 +1,58 @@
+include::ROOT:partial$variables.adoc[]
+
+= Data Privacy Console
+
+The Data Privacy Console enables you to access the overview of your personal data, request data deletion or data export.
+
+This feature allows all users to browse locations and contents of their private data and make requests concerning the treatment of this data (e.g. Delete, Update), which are then escalated to the <<data-privacy-officer, *data privacy officer (DPO)*>> Console.
+
+.The screenshot shows the overview of the Data Privacy Console
+image::data-privacy/data-privacy-console.png[role="data-zoomable"]
+
+== Features of the Privacy Console
+
+* Data Overview
+* Privacy Requests
+* Data Export
+* Data Deletion
+
+=== *Data Overview*
```

Comment:
Keep in mind that titles/sub-titles won't be bolded in the final document; so `=== Some *Text*` => `=== Some Text`